src/rosdiscover/observer/ros1.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+__all__ = ('ROS1Observer',)
+
+import typing
+from typing import Collection, Dict, Set
+
+from loguru import logger
+from roswire import AppInstance, ROS1
+from roswire.common import SystemState
+
+from .nodeinfo import NodeInfo
+from .observer import Observer
+from ..interpreter import NodeContext, ParameterServer, SystemSummary
+
+if typing.TYPE_CHECKING:
+    from ..config import Config
+
+_NODES_TO_FILTER_OUT = ('/rosout',)
+_TOPICS_TO_FILTER_OUT = ('/rosout', '/rosout_agg')
+_SERVICES_TO_FILTER_OUT = ('set_logger_level', 'get_loggers')
+
+
+class ROS1Observer(Observer):
+
+    def __init__(self, app: AppInstance, config: 'Config') -> None:
+        super().__init__(app, config)
+
+    def observe(self) -> SystemSummary:
+        """Observe the state of the running system and produce a summery of the archtiecture."""
+        nodecontexts: Set[NodeContext] = set()
+        with self._app_instance.ros1() as ros:
+            info = ros.state
+            nodes = self._transform_info(info)
```

Comment:
It's not immediately clear what `_transform_info` does. What transformation is taking place?

Reply:
Renamed to `_transform_state_to_nodeinfo`

Reply:
Have you pushed the changes? Or is GitHub misbehaving again?